Project.toml

```diff
@@ -4,20 +4,26 @@ authors = ["Christoph Ortner <christophortner@gmail.com> and contributors"]
 version = "0.0.3"
 
 [deps]
+Combinatorics = "861a8166-3701-5b0c-9a16-15d98fcdc6aa"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+PartialWaveFunctions = "793d2195-304b-438e-bbb1-bc33c872ac39"
+Polynomials4ML = "03c4bcba-a943-47e9-bfa1-b1661fc2974f"
```

Comment:
Why are P4ML and SpheriCart dependencies? It seems to me these are only test dependencies?

Reply:
Right... I think the only two new packages needed are PartialWaveFunc and Combinatorics, all others should be just for tests. Will do a cleanup.